server/api/models/member.js

```diff
@@ -0,0 +1,49 @@
+module.exports = {
+  attributes: {
+    id: {
+      type: 'string',
+      columnName: '_id'
+    },
+    createdAt: {
+      type: 'number',
+      autoCreatedAt: true,
+    },
+    updatedAt: {
+      type: 'number',
```

Comment:
probably don't these 2 columns above to be explicitly defined since already defined in server/config/models.js